### DISCUSSION TITLE
Persistent issue on mvn clean install to load connector locally

### DISCUSSION THREAD

**Discussion by Drashti065** [MuleSoft-AI-Chain-Project/mule-ai-chain-connector#56] [Q&A]
Hello experts, 
I am trying my hands on this repo first time quite excited with few interesting use case. When I do mvn clean install, it throws me long error from /m2/repository
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-jdeps-plugin:3.1.2:jdkinternals (default) on project mulechain-ai-connector:
[ERROR] Exit code: 1 - The command line is too long.

any workaround for command too long? I am using windows.
I have latest maven installed. Any guidance to do check will be useful. 
I tired clearing repository 
using java8

Thank you 
Drashti

**Comment by amirkhan-ak-sf** [comment]
Hi Drashti

Check this: https://github.com/MuleSoft-AI-Chain-Project/mulesoft-ai-chain-connector/issues/33#issue-2477080722

**Comment by amirkhan-ak-sf** [comment]
Known issue > workaround: https://github.com/MuleSoft-AI-Chain-Project/mulesoft-ai-chain-connector/issues/33#issue-2477080722
